distributor.php

```diff
@@ -4,7 +4,7 @@
  * Plugin URI:        https://github.com/10up/distributor
  * Update URI:        https://distributorplugin.com
  * Description:       Makes it easy to distribute and reuse content across your websites, whether inside of a multisite or across the web.
- * Version:           1.6.5-dev
+ * Version:           1.6.5
```

Comment:
Build process strips this, `-dev` should remain here.

Reply:
Oops, you're right, release instructions even say that, my bad, nice catch!